.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ e2e-defaults: &e2e-defaults
   docker:
     - image: celohq/circleci
   environment:
-    CELO_BLOCKCHAIN_BRANCH_TO_TEST: master 
+    CELO_BLOCKCHAIN_BRANCH_TO_TEST: victor/gateway-fee
```

Comment:
TODO revert?

Reply:
My plan is to merge https://github.com/celo-org/celo-blockchain/pull/602, then revert this, then merge this PR. Thanks for making sure I didn't miss this.